### DISCUSSION TITLE
Kite Model

### DISCUSSION THREAD

**Discussion by jeremy-rutman** [JSBSim-Team/jsbsim#535] [Q&A]
Hi , I'm interested in modeling a certain kite.  Let's assume for the time being that it's rigid and doesn't deform, if I define the geometry e.g. in a cad program is this enough to be able to use this kite in jsbsim, or do I need e.g. a set of lift and drag coefficients per wind speed and angle?  
Also - can jsbsim deal with a tether adding a new force to the aircraft , and furthermore forces on the tether itself.  
Apologies for the noobiness of the question .

**Comment by seanmcleod** [comment]
JSBSim can't take a CAD model/geometry as input, X-Plane allows you to do that.

Take a look at the JSBSim Reference Manual - https://jsbsim.sourceforge.net/JSBSimReferenceManual.pdf.
